app/models/dto.py

```
@@ -67,6 +67,75 @@ class AvailabilityRequest(BaseModel):
+    @model_validator(mode='after')
+    def validate_logic(self) -> 'AvailabilityRequest':
+        # 1. Coordinate Logic
+        if self.swLat >= self.neLat:
+            raise ValueError("남서쪽 위도(swLat)는 북동쪽 위도(neLat)보다 작아야 합니다.")
+        if self.swLng >= self.neLng:
+            raise ValueError("남서쪽 경도(swLng)는 북동쪽 경도(neLng)보다 작아야 합니다.")
+            
+        # 2. Time Logic (Start < End & Past Time)
+        from datetime import datetime, date
+        # 2. Time Logic (Start < End & Past Time)
+        from datetime import datetime, date
```

Ruff (0.15.1)

[error] Redefinition of unused `datetime` from line 118: `datetime` redefined here

Remove definition: `datetime`

(F811)